### PR TITLE
Create Ananya_ksp.md

### PR DESCRIPTION
**Pull request by Ananyaiitbhilai**
my blog for ksp

### CODE REVIEW

**Comment by Ananyaiitbhilai** [comment]
my gif in readme is not showing

**Comment by Lalit3716** [comment]
> my gif in readme is not showing

You forgot to add quotation marks around width=40%
In img tag. Try doing that

**Comment by Ananyaiitbhilai** [comment]
> > my gif in readme is not showing
> 
> You forgot to add quotation marks around width=40% In img tag. Try doing that

i tried this but still not coming however in review it is showing

**Review by ExpressHermes — APPROVED** [review]
LGTM!

**Review by ashutosh321607 — APPROVED** [review]

File: content/blog/Ananya_ksp.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+## 👩🏻‍💻 My motivation for the project
+
+I chose this project because of the following reason, I myself sometimes contact multiple people on Chat platforms to get feedback about a course and its material/resources. Crowd-sourcing of materials and feedback would make it easy to organize resources at one place. It would be more reliable, save one’s time and avoid unnecessary bothering. Finding this project solves a real-life problem and relatable too, I was motivated for the project throughout.
+
+## 🗒 My Work
+
+Discussing the work I did during the period, I mainly worked on designing the whole backend of the project and I also helped in designing and adding some frontend components. I used django restframework, and designed the models, used axios to communicate with backend and coded APIs for "GET" and "POST" requests. <br>
+
+## 📉 How Fossoverflow helped in my overall development
```
Comment (ambarvm) [inline content/blog/Ananya_ksp.md:19]:
```suggestion
## 📈 How Fossoverflow helped in my overall development
```

**Pull request merged**